guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,43 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+.Procedure
+
+. In the {Project} web UI, click *Administer* > *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to reveal a list of available events.
+. Select an event from the list.
+. Enter a name.
+. Enter a target URL. When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
+. Click *Template* to see the list of available templates.
+. Select a template from the list.
+. Enter a HTTP method.
+. Ensure that the *Enabled* flag is checked if you want to create active webhook.
+
+In the Credentials tab:
+
+. Optional: Enter the username and password when HTTP authentication is required. Do not enter anything when using shellhooks.
+. Select *Verify SSL* if the server certificate should be verified against the system certificate store or Foreman CA. When using shellhooks, ensure this flag is selected.
+. Select *Proxy Authorization* when using shellhooks, otherwise deselect this box.
+. Optional: Provide one or more CA certificates in the X509 Certification Authorities. Keep this field blank when using shellhooks.
+
+In the Additional tab:
+
+. Enter the HTTP Content Type, typically application/json, application/xml or text/plain depending on the payload.
+. Optional: Provide HTTP headers as JSON. ERB is allowed and the same macros and variables are available as in webhook templates.
+
+When configuring webhooks with endpoints with non-standard HTTP(s) ports, SELinux port must be assigned, see  https://access.redhat.com/documentation/en-us/red_hat_satellite/6.9/html-single/installing_satellite_server_from_a_connected_network/index#configuring-selinux-to-ensure-access-on-custom-ports_satellite[Configuring SELinux to Ensure Access to Satellite on Custom Ports] in _Installing Satellite Server From A Connected Network_.
```

Comment:
```suggestion
When configuring webhooks with endpoints with non-standard HTTP(s) ports, SELinux port must be assigned, see  https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_connected_network/index#configuring-selinux-to-ensure-access-on-custom-ports_satellite[Configuring SELinux to Ensure Access to Satellite on Custom Ports] in _Installing Satellite Server From A Connected Network_.
```

Additionally, use an asciidoc attribute to link to access.redhat.com. See [attributes L17](https://github.com/theforeman/foreman-documentation/blame/master/guides/common/attributes.adoc#L17)

Reply:
He should just be using attributes L17 rather than adding {AccessRedHatComVersion} in this case. That is for Red Hat URLs that are not upstream guides, if I understand correctly.

Reply:
My bad. If the content is available upstream, then yes. It's probably something like `{InstallingProjectDocURL}`.